almdrlib/client.py

```diff
@@ -124,7 +124,7 @@ def validate(self, data):
             self._validator.validate(data)
         except jsonschema.exceptions.ValidationError as e:
             raise AlmdrlibValueError(
-                    f"{e.message}. Schema: {json.dumps(e.schema)}")
+                    f"Validation error: {e.message}.\nSchema:\n{json.dumps(e.schema, indent=2)}")
```

Comment:
perhaps we should print a github link to the schema and or introduce a verbose flag and dump out the schema only when`verbose` is set?

Reply:
It is a good idea, though we would need to elaborate how to pull the link and make it point to the particular place in the schema, as it might be non-trivial. If you are talkin about general link to yaml, not sure how could it be useful. Maybe we can just suggest to run with -v if there is a desire to see the schema dump.

Reply:
+1, if generating a link to the correct schema version is difficult then let's go with the `-v` to enable the entire error message.

Reply:
I updated it to use logging.debug for the schema.  No `-v` needed, just `--debug` in alcli.